source/Service.js

```diff
@@ -124,6 +125,7 @@ class Service extends EventEmitter {
         this._helpers = [];
         this._initialised = false;
         this._shutdown = false;
+        this._logger = new Logger(storage);
```

Comment:
I would suggest passing `this` as the parameter instead of storage, as having a reference is a good idea for event listeners..